### PR TITLE
LPS-118113 FAKE GRADLE CACHE

### PR DESCRIPTION
**Pull request by lawrence-lee**
Please publish `portal-tools-bundle-support`, then `gradle-plugins-workspace`, then `project-templates-extensions`, then `project-templates`. Thanks!

### CODE REVIEW

**Comment by lawrence-lee** [comment]
ci:test:sf

**Comment by lawrence-lee** [comment]
ci:test:project-templates

File: modules/sdk/project-templates/project-templates-workspace/src/main/resources/archetype-resources/gradle.properties [hunk 1]
```diff
@@ -80,6 +80,12 @@
     #
     #liferay.workspace.modules.jsp.precompile.enabled=false
 
+    #
+    # Set this property to build Node.js-style projects using Yarn. The default
+    # value is npm.
+    #
+    #liferay.workspace.node.package.manager=yarn
```
Comment (gamerson) [inline modules/sdk/project-templates/project-templates-workspace/src/main/resources/archetype-resources/gradle.properties:87]: hey @lawrence-lee  is this the default?

File: modules/sdk/project-templates/project-templates-workspace/src/test/java/com/liferay/project/templates/workspace/ProjectTemplatesWorkspaceTest.java [hunk 2]
```diff
@@ -187,6 +187,42 @@ public void testBuildTemplateWorkspaceLocalProperties() throws Exception {
...
+			mavenExecutor);
+
+		File gradleProperties = new File(
+			workspaceProjectDir, "gradle.properties");
+
+		Assert.assertTrue(gradleProperties.exists());
+
+		String configLine =
+			System.lineSeparator() +
+				"liferay.workspace.node.package.manager=yarn";
```
Comment (gamerson) [inline modules/sdk/project-templates/project-templates-workspace/src/test/java/com/liferay/project/templates/workspace/ProjectTemplatesWorkspaceTest.java:205]: If `yarn` is the default, do we need a test with this set to `npm`?

File: modules/sdk/project-templates/project-templates-workspace/src/main/resources/archetype-resources/gradle.properties (hunk 1 above)
Comment (lawrence-lee) [reply modules/sdk/project-templates/project-templates-workspace/src/main/resources/archetype-resources/gradle.properties:87]: npm is default. Thats why i worded the blurb the way i did. I thought it was the shortest most concise way to saying there are 2 options `npm` and `yarn`

**Comment by lawrence-lee** [comment]
ci:test:project-templates

**Comment by gamerson** [comment]
hey @petershin  this is LGTM! 👍

**Comment by lawrence-lee** [comment]
ci:test:sf

**Comment by lawrence-lee** [comment]
ci:test:project-templates

**Comment by lawrence-lee** [comment]
ci:test:project-templates

**Comment by petershin** [comment]
ci:forward

**Pull request closed without merging**